doc/source/guidelines/doc_practices.rst

```diff
@@ -0,0 +1,352 @@
+
+#. Type ``ls`` to see a list of the files now in your local directory. 
+
+.. note::
+   In Anaconda Powershell, typing ``dir`` is the equivalent to typing ``ls``.
+
+Using GitBash to Clone a GitHub Repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#. In the folder where you clone PyAnsys repositories, right-click and select
+   ``GitBash Here``.
+
+#. Type ``git clone`` followed by the address of the repo to clone.
+
+For example, to clone the ``pyansys-sphinx-theme`` repository, you would
+enter:
+  
+.. code::
+  
+  git clone https://github.com/pyansys/pyansys-sphinx-theme
+
+.. _create_pr:
+
+Pushing Changes to the GitHub Repository
+----------------------------------------
+Once you have tested your changes in local documentation builds and are
+satisifed with them, you can push them to the library's GitHub repository. 
+You can use Git commands or Git Extension to accomplish this. 
+
+For documentation changes, the branch name should have a prefix of ``doc\``
+followed by a short description of what you've changed. For more information, see
+:ref:`branch_naming`.
+
+Using Git Commands to Push Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This procedure assumes that you are using Git Commands to push your changes to the
+GitHub repository.
+
+#. In Windows Command Prompt, navigate to the directory where you cloned 
+   the GitHub reposity.
+
+#. Type ``git status`` to see a summary of the changes that you have made in this
+   directory.
+
+#. Create a branch by entering ``git checkout -b`` followed by the name to give
+   this branch.
+   
+   The option ``-b`` is for adding a branch. If you were adding a branch with documentation
+   changes to the ``Contributing`` topic, you might enter:
+   
+   .. code::
+
+    git checkout -b edit_contributing
+   
+#. To add and check out this new branch, enter ``git add checkout -b doc/`` followed by
+   the branch name that you specified in the previous step.
+
+#. Enter ``git add .`` to stage the changed files to commit.
+
+#. Enter ``git status`` again to see all files that are staged for committing in green.
+
+#. If any unnecessary files are staged, delete them in File Explorer and then enter ``git status`` 
+   again to ensure that the final red line indicates that these files have been deleted.
+
+#. If you make any additional file changes, type ``git add .`` again to restage the files to
+   commit.
+
+#. When finished, commit your changes to GitHub history by entering ``git commit -m`` followed by
+   a description in quotation marks.
+   
+   For the given example, you might enter:
+   
+   .. code::
+
+    git commit -m "Edit Contributing topic" 
+   
+   The number of files changed are shown, followed by the number of insertions and deletions.
+   A create mode or rename doc action is then shown for each file.
+
+#. Enter ``git push``, which shows a message like this to indicate that changed files have not
```

Comment:
Putting a bare `git push` here doesn't really make sense to me, since the command has no effect. 

What we need to communicate here is that the _first_ `git push` after a new branch has been created should be 

``git push --set-upstream origin doc/edit_contributing``

Any subsequent pushes can be simply ``git push``.

Maybe it would be better to flip the order of commands, and mention that the ``--set-upstream`` command only needs to be executed the first time?